geomesa-quickstart-hbase/src/main/java/com/example/geomesa/hbase/HBaseQuickStart.java

```diff
@@ -0,0 +1,249 @@
+        for (String param : HBASE_CONNECTION_PARAMS) {
+            dsConf.put(param.replace("_", "."), cmd.getOptionValue(param));
+        }
+        return dsConf;
+    }
+
+    static SimpleFeatureType createSimpleFeatureType(String simpleFeatureTypeName)
+            throws SchemaException {
+
+        // list the attributes that constitute the feature type
+        List<String> attributes = Lists.newArrayList(
+                "Who:String",
+                "What:java.lang.Long",     // some types require full qualification (see DataUtilities docs)
+                "When:Date",               // a date-time field is optional, but can be indexed
+                "*Where:Point:srid=4326",  // the "*" denotes the default geometry (used for indexing)
+                "Why:String"               // you may have as many other attributes as you like...
+        );
+
+        // create the bare simple-feature type
+        String simpleFeatureTypeSchema = Joiner.on(",").join(attributes);
+        SimpleFeatureType simpleFeatureType =
+                DataUtilities.createType(simpleFeatureTypeName, simpleFeatureTypeSchema);
+
+        return simpleFeatureType;
+    }
+
+    static FeatureCollection createNewFeatures(SimpleFeatureType simpleFeatureType, int numNewFeatures) {
+        DefaultFeatureCollection featureCollection = new DefaultFeatureCollection();
+
+        String id;
+        Object[] NO_VALUES = {};
+        String[] PEOPLE_NAMES = {"Addams", "Bierce", "Clemens"};
+        Long SECONDS_PER_YEAR = 365L * 24L * 60L * 60L;
+        Random random = new Random(5771);
+        DateTime MIN_DATE = new DateTime(2014, 1, 1, 0, 0, 0, DateTimeZone.forID("UTC"));
+        Double MIN_X = -78.0;
+        Double MIN_Y = -39.0;
+        Double DX = 2.0;
+        Double DY = 2.0;
+
+        for (int i = 0; i < numNewFeatures; i ++) {
+            // create the new (unique) identifier and empty feature shell
+            id = "Observation." + Integer.toString(i);
+            SimpleFeature simpleFeature = SimpleFeatureBuilder.build(simpleFeatureType, NO_VALUES, id);
+
+            // be sure to tell GeoTools explicitly that you want to use the ID you provided
+            simpleFeature.getUserData().put(Hints.USE_PROVIDED_FID, java.lang.Boolean.TRUE);
+
+            // populate the new feature's attributes
+
+            // string value
+            simpleFeature.setAttribute("Who", PEOPLE_NAMES[i % PEOPLE_NAMES.length]);
+
+            // long value
+            simpleFeature.setAttribute("What", i);
+
+            // location:  construct a random point within a 2-degree-per-side square
+            double x = MIN_X + random.nextDouble() * DX;
+            double y = MIN_Y + random.nextDouble() * DY;
+            Geometry geometry = WKTUtils$.MODULE$.read("POINT(" + x + " " + y + ")");
+
+            // date-time:  construct a random instant within a year
+            simpleFeature.setAttribute("Where", geometry);
```

Comment:
This line ought to be line 134.  Yet another item I got wrong in the original.  Sorry.